zebra-chain/src/test/vectors.rs

```diff
@@ -0,0 +1,10 @@
+//! Test vectors for blockchain contructions
+use hex::FromHex;
+
+use lazy_static::lazy_static;
+
+lazy_static! {
```

Comment:
Question: these test vectors don't depend on zebra-chain. Would they be more useful in zebra-test?